x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -117,25 +117,27 @@ List every hostname and variant used to connect to your cluster over HTTPS.
 ----
 --
 
-. Copy the relevant `http.p12` certificate to the `ES_PATH_CONF` directory on each node.
+. On *every* node in your cluster, complete the following steps:
 
-. On each node, edit the `elasticsearch.yml` file to enable HTTPS security and
-   specify the location of the `http.p12` security certificate.
+  .. Copy the relevant `http.p12` certificate to the `$ES_PATH_CONF` directory.
+
+  .. Edit the `elasticsearch.yml` file to enable HTTPS security and specify the
+  location of the `http.p12` security certificate.
 +
 [source,yaml]
 ----
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.keystore.path: http.p12
+xpack.security.http.ssl.keystore.path: config/http.p12
```

Comment:
The `config/` part is not needed. Configuration files are resolved relative to the config directory.

Reply:
Thanks @ywangd -- I'll remove `config/` from the path.